cli/tests/test_git_import_export.rs

```diff
@@ -261,13 +272,22 @@ fn get_bookmark_output(test_env: &TestEnvironment, repo_path: &Path) -> String {
     test_env.jj_cmd_success(repo_path, &["bookmark", "list", "--all-remotes"])
 }
 
-fn get_git_repo_refs(git_repo: &git2::Repository) -> Vec<(String, CommitId)> {
+fn get_git_repo_refs(git_repo: &gix::Repository) -> Vec<(String, CommitId)> {
     let mut refs: Vec<_> = git_repo
         .references()
         .unwrap()
-        .filter_ok(|git_ref| git_ref.is_tag() || git_ref.is_branch() || git_ref.is_remote())
-        .filter_map_ok(|git_ref| {
-            let full_name = git_ref.name()?.to_owned();
+        .all()
+        .unwrap()
+        .filter_ok(|git_ref| {
+            matches!(
+                git_ref.name().category(),
+                Some(gix::reference::Category::Tag)
+                    | Some(gix::reference::Category::LocalBranch)
+                    | Some(gix::reference::Category::RemoteBranch),
+            )
+        })
+        .filter_map_ok(|mut git_ref| {
+            let full_name = git_ref.name().as_bstr().to_string();
```

Comment:
nit: this could be `.as_bstr().to_owned()` (i.e. return `BString`) instead of doing lossy conversion here.